index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div id="main">
+<div id="title-div">
+    <h1 id="title">Dr Norman Borlaug</h1>
+    <p id="saver"> The man who saved a billion lives</p>
+</div>
+<div id="img-div">
+    <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <div id="tribute-link"></div>
+    <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+</div>
+    <h3 id="tribute-info">Here's a time line of Dr. Borlaug's life:</h3>
+    <div id="timeline-div" style="text-align: center;">
```

Comment:
I think you should use external CSS instead of inline style because the use of external CSS is a professional way to give style properties to HTML elements.

Reply:
- [ ] Very nice usage of inline comments. please keep it up while doing reviews for your partners and colleagues.